RELEASING/README.md

```diff
@@ -69,7 +103,7 @@ set your GITHUB_TOKEN environment variable.
 
 ```bash
 # will overwrites the local CHANGELOG.md, somehow you need to merge it in
-github-changes -o apache -r incubator-superset --token $GITHUB_TOKEN --between-tags <PREVIOUS_RELEASE_TAG>...<CURRENT_RELEASE_TAG>
+github-changes -o apache -r incubator-superset --token $GITHUB_TOKEN -b $SUPERSET_GITHUB_BRANCH
```

Comment:
Had conversations with @dpgaspar before about `github-changes` struggling with cherries, probably because the SHAs are different on different branches... Does `-b` help?

I thought we may need to do something more sophisticated in the future using PR numbers extracted from commit messages...

Reply:
It seems `-b` solved all my problems (output completely in line with the log from `git` but with additional github context), so cautiously optimistic about being able to keep using `github-changes`.

Reply:
fingers crossed here. Problems may arise on a new minor, 0.36 for example, where we need to diff with master and the SHA's changed